dask_ml/model_selection/_incremental.py

```diff
@@ -476,26 +476,55 @@ def _get_history_results(self, results):
         * partial_fit_calls
         * model_id
         """
-        info, model, history = results
+        _, model, history = results
+        info = {}
         key = operator.itemgetter("model_id")
         hist2 = sorted(history, key=key)
-        return hist2
-
-    def _get_best(self, results, history_results):
-        # type: (Dict, Dict) -> Estimator
-        """Select the best estimator from the set of estimators."""
-        best_model_id = first(results.info)
-        key = operator.itemgetter("model_id")
-        best_index = -1
-        # history_results is sorted by (model_id, partial_fit_calls)
-        # best is the model_id with the highest partial fit calls
-        for k, v in itertools.groupby(history_results, key=key):
-            v = list(v)
-            best_index += len(v)
-            if k == best_model_id:
-                break
+        for h in hist2:
+            if h["model_id"] not in info:
+                info[h["model_id"]] = []
+            info[h["model_id"]] += [h]
+        return hist2, info
+
+    def LD_to_DL(self, LD):
+        """ list of dicts to dict of lists. Assumes same keys in all dicts.
+        https://stackoverflow.com/questions/5558418/list-of-dicts-to-from-dict-of-lists
+        """
+        return {k: [dic[k] for dic in LD] for k in LD[0]}
 
-        return results.models[best_model_id], best_index
+    def _get_cv_results(self, history, model_hist):
+        """
+        history is output of _get_history_results; same keys
+        """
+        cv_results = {}
+        best_scores = {}
+        for h in history:
+            key = h["model_id"]
+            pf_times = list(toolz.pluck("partial_fit_time", model_hist[key]))
+            score_times = list(toolz.pluck("score_time", model_hist[key]))
+            best_scores[key] = h["score"]
+            scores = np.array(list(best_scores.values()))
+            cv_results[key] = {
+                "mean_partial_fit_time": np.mean(pf_times),
+                "mean_score_time": np.mean(score_times),
+                "std_partial_fit_time": np.std(pf_times),
+                "std_score_time": np.std(score_times),
+                "test_score": h["score"],
+                "rank_test_score": sum(scores >= h["score"]),
+                "model_id": h["model_id"],
+                "params": h["params"],
+                "partial_fit_calls": h["partial_fit_calls"],
+            }
+        cv_results = list(cv_results.values())  # list of dicts
+        cv_results = self.LD_to_DL(cv_results)
+        cv_results = {k: np.array(v) for k, v in cv_results.items()}
+        return cv_results
```

Comment:
What does each record here indicate? For e.g. `GridSearchCV`, looking at the first element from each list (so `pd.DataFrame(cv_results).iloc[0]`) is the all the stats for a single hyper parameter combination.

Ours would have the same meaning, correct?

Reply:
Correct. `cv_results_` holds certain variables (`params`, `test_score`, `mean_partial_fit_time`, etc) for a particular parameter combo. All time-dependent variables (`test_score`, etc) are for the last time the model is evaluated.

Reply:
Good deal. In that case, it would be good to have this available to users.